chapters/0900 Statements.md

```diff
@@ -17,31 +78,317 @@ is intended to describe aspects of Swift analogous to what's found in
+
+##### Case Condition
+
+<div class="admonition grammar">
+	<p class="admonition-title">Grammar of a case condition</p>
+	<div class="syntax-group">
+		<p class="syntax-def">
+			<span class="name">case-condition</span>
+			<span class="arrow"> → </span> 
+			<code>case</code>
+			<a href="/inactiveLink">pattern</a>
+			<a href="/inactiveLink">initializer</a>
+		</p>
+	</div>
+</div>
+
+<p class="item"> </p>
+A case condition yields `true` if the initialization succeeds (i.e., the  
+pattern is matched), and otherwise yields `false`.
+
+*...to be continued...*
+
+##### Availability Condition
+
+An availability condition is evaluated at run time to determine whether given 
+API  are available based on the platform and platform version on which a program 
+is running.
+
+<div class="admonition grammar">
+	<p class="admonition-title">Grammar of an availability condition</p>
+	<div class="syntax-group">
+		<p class="syntax-def">
+			<span class="name">availability-condition</span>
+			<span class="arrow"> → </span> 
+			<code>#available</code>
+			<code>(</code>
+			<a href="/inactiveLink">availability-arguments</a>
+			<code>)</code>
+		</p>
+	</div>
+</div>
+
+<p class="item"> </p>
+An availability condition yields `true` if one of the platforms that is the 
+subject of the availability condition is present, and otherwise yields `false`.
+
+*...to be continued...*
+
 #### Branch Statements
+
+<div class="admonition grammar">
+	<p class="admonition-title">Grammar of a branch statement</p>
+	<div class="syntax-group">
+		<p class="syntax-def">
+			<span class="name">branch-statement</span>
+			<span class="arrow"> → </span> 
+			<a href="#if-statement">if-statement</a>
+			<code>;</code><sub>opt</sub>
+		</p>
+		<p class="syntax-def">
+			<span class="name">branch-statement</span>
+			<span class="arrow"> → </span> 
+			<a href="#guard-statement">guard-statement</a>
+			<code>;</code><sub>opt</sub>
+		</p>
+		<p class="syntax-def">
+			<span class="name">branch-statement</span>
+			<span class="arrow"> → </span> 
+			<a href="#switch-statement">switch-statement</a>
+			<code>;</code><sub>opt</sub>
+		</p>
+	</div>
+</div>
+
+##### If Statement
+<div class="admonition grammar">
+	<p class="admonition-title">Grammar of an if statement</p>
+	<div class="syntax-group">
+		<p class="syntax-def">
+			<span class="name">if-statement</span>
+			<span class="arrow"> → </span>
+			<code>if</code>
+			<a href="condition-list">condition-list</a>
+			<a href="/inactiveLink">code-block</a>
+			<a href="#else-clause">else-clause</a><sub>opt</sub>
+		</p>
+		<p class="syntax-def">
+			<a id="else-clause"></a>
+			<span class="name">else-clause</span>
+			<span class="arrow"> → </span> 
+			<code>else</code>
+			<a href="/inactiveLink">code-block</a>
+			|
+			<code>else</code>
+			<a href="#if-statement">if-statement</a>
+		</p>
+	</div>
+</div>
+
+<p class="item"> </p>
+The `if` keyword is dependent upon the `Bool` type declared in the standard 
```

Comment:
The meaning of this statement is very non-obvious to me.  Is it relevant where `Bool` is declared?  In what sense can a keyword (syntactic element) be “dependent upon” a type?

Reply:
One of the unusual (IMO) things about Swift is the core language's dependency upon the standard library.  Things like `if` and `for`-`in` loops simply cannot work without certain types vended by the standard library.  Where that occurs, it seems worth noting.